### DISCUSSION TITLE
EdgeTX on Horus X12S with develop board pcbdev

### DISCUSSION THREAD

**Discussion by jebawdej** [EdgeTX/edgetx#3365] [Q&A]
Hello, I had installed EdgeTX V2.8.1 on my Jumper T16 pro and I love it. Now I try to install this version on my Horus X12S using companion 2.8.1. But I can only choose a firmware file from the collection. I have tried to install this but get the messagebox that the firmware is not compatible. I have one from the first releases of X12S (with developer board) Normaly I must set the pcbdev option in the settings menu of companion. Is it possible to get a compatible firmware file?

**Comment by gagarinlg** [comment]
Getting customizer buolds via companion is not supported yet.
You can use gitpod:
https://github.com/EdgeTX/edgetx/wiki/Building-radio-firmware-in-a-webbrowser-with-Gitpod

**Reply by jebawdej** [reply]
OK thanks, where can I find how to build the firmware for the horus X12S and change the options?

**Comment by gagarinlg** [comment]
The example is for a TX16S. I think you have to change PCBREV to the correct value for your board.
I am not at my PC atm, so I can not check now

**Reply by jebawdej** [reply]
Looking to the compilation options, pcbdev is not yet supported. See: https://github.com/EdgeTX/edgetx/wiki/Compilation-options
